versions/3.1.1.md

```diff
@@ -563,7 +563,7 @@ my.org.User
   "securitySchemes": {
     "api_key": {
       "type": "apiKey",
-      "name": "api_key",
+      "name": "api-key",
```

Comment:
Here I have only changed the `name` and kept the `id` to be "api_key", therefore later in "Security Requirement Object" section, it stays using "api_key"
https://github.com/OAI/OpenAPI-Specification/blob/9bf45ceb368f3fe14d3f7e18ace19ce4b3f24ad5/versions/3.1.1.md?plain=1#L3370-L3381

I first replaced all `api_key` by `api-key` but I thought it would be better to keep consistent `api_key` to have consistent naming with `petstore_auth`